roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -87,8 +87,10 @@ kubernetesMasterConfig:
   - v1beta3
   - v1
 {% endif %}
-  apiServerArguments: {{ openshift.master.api_server_args | default(None) | to_json }}
-  controllerArguments: {{ openshift.master.controller_args | default(None) | to_json }}
+  apiServerArguments: 
+    {{ openshift.master.api_server_args | default(None) | to_yaml }}
+  controllerArguments: 
+    {{ openshift.master.controller_args | default(None) | to_yaml }}
```

Comment:
Have you tested this with controller_args that would expand further than one line? The main reason for embedding the json was to avoid having to deal with properly embedding the generated yaml.